signalfx-smart-gateway/README.md

```diff
@@ -43,7 +43,16 @@ The downloaded checksum assumes that the Smart Gateway binary is in the current
 
```

Comment:
Make sure that you also update the commands above:
the curl command under "Installation" and the checksum curl command both need updated versions